modules/installing-ocp-agent.adoc

```diff
@@ -127,6 +129,9 @@ sshKey: |
 You must provide the IP address when you do not specify the node's IP addresses in the `networkConfig` parameter. If this address is not provided, one IP address is selected from the provided nodes's `networkConfig`.
 <2> The number of hosts defined must match the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters. When 3 master nodes and 0 worker nodes are defined in the `install-config.yaml` file,
 the number of hosts defined is 3. When 3 master nodes and 2 worker nodes are defined in the `install-config.yaml` file, the number of hosts defined is 5.
+<3> The name of the host.
```

Comment:
Could mention that this is optional, and the hostname will be picked up from DHCP or reverse DNS if it is not supplied. But each host must have a hostname from at least one of those 3 sources (i.e. if you have no DHCP or reverse DNS set up, this _is_ required).